tests/RobotFramework/tests/cumulocity/registration/registration_lifecycle.robot

```diff
@@ -306,6 +306,7 @@ Early data messages cached and processed
     END
 
 Entities persisted and restored
+    Skip    Persistence is not a mapper responsibility anymore
     Execute Command    sudo tedge config set c8y.entity_store.clean_start false
     Restart Service    tedge-mapper-c8y
     Service Health Status Should Be Up    tedge-mapper-c8y
```

Comment:
Since it's registration_lifecycle test, I think the purpose of this test is still valid. Instead of skipping it, can't we replace tedge-mapper-c8y with tedge-agent?

```suggestion
    Execute Command    sudo tedge config set agent.entity_store.clean_start false
    Restart Service    tedge-agent
    Service Health Status Should Be Up    tedge-agent
```

Reply:
Resolved by d750fe